build-caching/src/main/java/com/vertispan/j2cl/build/Dependency.java

```diff
@@ -44,4 +61,52 @@ public Scope getScope() {
     public void setScope(Scope scope) {
         this.scope = scope;
     }
+
+    @Override
+    public boolean isAPT() {
```

Comment:
Docs should definitely point out that `isApt()` _must_ be called before `getProcessors()`, otherwise it will incorrectly return empty.